opensearch_dashboards.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "indexManagementDashboards",
-  "version": "1.0.0.0-rc1",
-  "opensearchDashboardsVersion": "1.0.0-rc1",
+  "version": "1.0.0.0",
```

Comment:
1.0.0.0? If this is for 1.0.0 Dashboards then plugin should be 1.1.0.0

Reply:
Can bump this during the 1.1.0 release

Reply:
ok